GooglePlayInstant/Editor/PlayInstantAssetBundleUrlVerifierWindow.cs

```diff
@@ -0,0 +1,135 @@
+        }
+
+        //TODO: Support Unity 5.6.0+
+        private static void UpdateAssetBundleVerificationInfoWindow()
+        {
+            var www = UnityWebRequestAssetBundle.GetAssetBundle(_assetBundleUrl);
+            www.SendWebRequest();
+            while (!www.isDone)
+            {
+                //TODO: implement loading bar 
+            }
+
+            var bundle = DownloadHandlerAssetBundle.GetContent(www);
+
+            _responseCode = www.responseCode;
+
+            if (www.isNetworkError || www.isHttpError)
+            {
+                _assetBundleDownloadIsSuccessful = false;
+                _errorDescription = www.error;
+                Debug.LogErrorFormat("Problem retrieving AssetBundle from {0}: {1}", _assetBundleUrl,
+                    _errorDescription);
+            }
+            else if (bundle == null)
+            {
+                _assetBundleDownloadIsSuccessful = false;
+                _errorDescription = "Error extracting AssetBundle. See Console log for details.";
+                // No need to log since debugging information in this case is automatically logged by Unity.
+            }
+            else
+            {
+                _assetBundleDownloadIsSuccessful = true;
+                _numOfMegabytes = ConvertBytesToMegabytes(www.downloadedBytes);
+
+                var scenes = bundle.GetAllScenePaths();
+                _mainScene = (scenes.Length == 0) ? "No scenes in AssetBundle" : scenes[0];
+
+                // Free memory used by the AssetBundle since it will not be in use by the Editor. Set to true to destory
+                // all objects that were loaded from this bundle.
+                bundle.Unload(true);
+            }
+        }
+
+        private static double ConvertBytesToMegabytes(ulong bytes)
+        {
+            return bytes / 1024f / 1024f;
+        }
+
+        private void OnGUI()
+        {
+            
+            UpdateAssetBundleVerificationInfoWindow();
+
+            AddVerifyComponentInfo("AssetBundle Download Status:",
+                _assetBundleDownloadIsSuccessful ? "SUCCESS" : "FAILED");
+
+            AddVerifyComponentInfo("AssetBundle URL:", _assetBundleUrl, EditorStyles.wordWrappedLabel);
+
+            AddVerifyComponentInfo("HTTP Status Code:", _responseCode == 0 ? "N/A" : _responseCode.ToString());
+
+            AddVerifyComponentInfo("Error Description:", _assetBundleDownloadIsSuccessful ? "N/A" : _errorDescription);
+
+            AddVerifyComponentInfo("Main Scene:", _assetBundleDownloadIsSuccessful ? _mainScene : "N/A");
+
+            AddVerifyComponentInfo("Size (MB):",
+                _assetBundleDownloadIsSuccessful ? _numOfMegabytes.ToString("#.####") : "N/A");
+
+            if (GUILayout.Button("Refresh"))
+            {
+                UpdateAssetBundleVerificationInfoWindow();
+            }
+        }
+
+        private void AddVerifyComponentInfo(string title, string response, GUIStyle layout = null)
```

Comment:
Is there any harm in getting rid of the layout param and always using EditorStyles.wordWrappedLabel ? If yes, then this is fine. If no, then you could simplify things slightly by always calling with EditorStyles.wordWrappedLabel.

Reply:
Also delete ", GUIStyle layout = null"